### PR TITLE
Use update setup-gap action for gap v2

### PR DESCRIPTION
**Pull request by chainchad**
- [x] Merge https://github.com/smartcontractkit/.github/pull/693
- [x] Release version of setup-gap
- [x] Update ref to shasum once above is merged

### CODE REVIEW

File: actions/crib-deploy-environment/action.yml [hunk 1]
```diff
@@ -93,22 +76,17 @@ runs:
   using: "composite"
   steps:
     - name: Setup GAP
-      uses: smartcontractkit/.github/actions/setup-gap@d316f66b2990ea4daa479daa3de6fc92b00f863e # setup-gap@0.3.2
+      # TODO: change the git ref to the next version before merging.
+      uses: smartcontractkit/.github/actions/setup-gap@TODO # setup-gap@1.0.0
```
Comment (chainchad) [inline actions/crib-deploy-environment/action.yml:80]: TODO

File: actions/crib-deploy-environment/action.yml [hunk 2]
```diff
@@ -148,28 +125,15 @@ runs:
         sanitized_branch=$(echo "${{ steps.generate-ns-name.outputs.branch-name }}" \
           | sed 's/[^a-zA-Z0-9._-]/_/g' | cut -c1-63 | sed 's/[._-]$//')
 
-        # Validate that chainlink-team and chainlink-product inputs adhere to the Kubernetes label format.
```
Comment (njegosrailic) [inline actions/crib-deploy-environment/action.yml:151]: You may need to rebase the branch.

Comment (chainchad) [reply actions/crib-deploy-environment/action.yml:151]: Rebased/pushed but I still see this comment 🤔

Comment (chainchad) [reply actions/crib-deploy-environment/action.yml]: Thanks! Fixed in 63d54bb824334f6f7537831c8d12a70905d8f28a.

**Review by njegosrailic — APPROVED** [review]

**Pull request merged**